src/components/FoodList/FoodList.js

```diff
@@ -1,28 +1,51 @@
-import foodData from '../../data/foodData.json'
 import styled from 'styled-components/macro'
+import { useState, useEffect } from 'react'
+import getFood from '../../services/getFood'
 
 export default function FoodList() {
+  const [foodList, setFoodList] = useState([])
+  const [searchInput, setSearchInput] = useState('')
+
+  useEffect(() => {
+    getFood()
+      .then((data) => setFoodList(data))
+      .catch((error) => console.log(error))
+  }, [])
+
+  function handleChange(event) {
+    setSearchInput(event.target.value)
+  }
+
+  const filteredFoodList = foodList.filter((food) =>
+    food.food.toLowerCase().includes(searchInput.toLowerCase())
+  )
+
   return (
-    <FoodListStyled>
-      {foodData.map((food) => (
-        <FoodListItem>{food.food}</FoodListItem>
-      ))}
-    </FoodListStyled>
+    <>
+      <form>
+        <SearchBar
+          type="text"
+          placeholder="suchen..."
+          onChange={handleChange}
+        />
+      </form>
+      <FoodListStyled>
+        {filteredFoodList.map(({ id, food }) => (
+          <FoodListItem key={id}>{food}</FoodListItem>
+        ))}
+      </FoodListStyled>
+    </>
   )
 }
 
 const FoodListStyled = styled.ul`
   margin: 0 20px;
   box-shadow: 0 0 10px #767670;
```

Comment:
You could define some variables for your colors instead of using the hex code. It will be more explicit.